ereolen_support/ereolen_material.form.inc

```diff
@@ -41,6 +41,17 @@ function ereolen_material_form($form, &$form_state) {
     '#required' => TRUE,
   ];
 
+  $form['ereolen_material_type'] = [
+    '#type' => 'select',
+    '#title' => t('Type'),
+    '#options' => [
+      t('E-Book') => t('E-Book'),
```

Comment:
“E-Book” is usually written as ”E-book” (cf. https://github.com/eReolen/base/blob/develop/sites/all/modules/reol_base/reol_base.module#L719).

Reply:
Changed